composer.json

```diff
@@ -2,7 +2,7 @@
   "name": "sinso/translationapi",
   "description": "API to fetch translation units.",
   "type": "typo3-cms-extension",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
We need to drop that version information from composer since it is best-practice to infer it from git tag instead.